client/login/wp-login/index.jsx

```diff
@@ -96,30 +96,41 @@ export class Login extends React.Component {
 		return <LocaleSuggestions locale={ this.props.locale } path={ this.props.path } />;
 	}
 
+	renderFooter() {
+		return (
+			<div className="wp-login__jetpack-footer">
+				<img src="/calypso/images/jetpack/powered-by-jetpack.svg" style={ { width: '200px' } } />
```

Comment:
This is how it's done in wp-login.php, but I agree it doesn't make much sense. I'll update.